### DISCUSSION TITLE
Welcome to WEB-STUDY Discussions!

### DISCUSSION THREAD

**Discussion by jandy14** [jandy14/WEB-STUDY#3] [일반]
## 👋 어서오세요!
  Discussions는 정리한 내용에 대한 궁금증에 대한 내용을 질문하기 위해 만들었습니다!

너무 귀찮게 하시면 닫아버릴거에요!

**Comment by rockpell** [comment]
레디스 정리 자료 잘 보고 갑니다~
Sentinel이나 cluster는 단일 서버 내에서도 좋은 효과를 볼 수 있겠지만 많은 수의 서버를 사용해야 의미가 있을 것 같아 보이네요.
아직 저한테는 먼 얘기 같습니다 ㅎㅎ

**Reply by jandy14** [reply]
둘다 단일 서버에서 사용한다면 그다지 효과가 없다고 생각합니다. 둘을 이용하는 이유가 로드밸런싱과 고가용성인데 단일서버는 로드밸런싱도 의미가 없고 서버에 장애가 일어나면 모든 프로세스가 죽기때문에 고가용성도 제공하지 못하게 됩니다.
